test/commands/entities/configure.test.ts

```diff
@@ -90,6 +96,7 @@ describe('entities:configure', () => {
     )
     .stdout()
     .command(['entities:configure',
+      '--data-type', 'not-set',
```

Comment:
Why were these additions necessary?

Reply:
I made changes to the test data. dataType ended up being the property that is getting configured, so it appears in the body and in the flags so it matches.